docs/reference/mpremote.rst

```diff
@@ -0,0 +1,90 @@
+.. _mpremote_files:
+
+MicroPython .mpremote files
+===========================
+
+This CLI tool provides an integrated set of utilities to remotely interact with and automate a MicroPython device over a serial connection.
+
+The simplest way to use this tool is:
+
+mpremote
```

Comment:
This needs to be indented to format it as code.  Also prefix a $ because it's at the prompt.

Reply:
> This needs to be indented to format it as code.

And `::` after "is" instead of just one `:`, IIRC.

Reply:
this still needs to be done